### PR TITLE
Adding extra conditions for match report layout in order to update ordering of components

### PR DESCRIPTION
**Pull request by rcrphillips**
## What does this change?
Adds a new layout of match report in order to have better control over the template. This helped with re-ordering the specific components on load for the template. Note that there are other visual changes in the upcoming match tabs PR https://github.com/guardian/dotcom-rendering/pull/1820.

### Before
Screen Shot 2020-08-18 at 16 34 44

### Frontend
Screen Shot 2020-08-18 at 16 34 49

### After
Screen Shot 2020-08-18 at 16 37 44

## Why?
Visual parity with frontend in order to help release match reports into DCR

### CODE REVIEW

File: src/web/layouts/StandardLayout.tsx [hunk 1]
```diff
@@ -52,108 +52,125 @@ import { Display } from '@root/src/lib/display';
...
-                        'lines'
-                        'meta'
-                        'body';
-                }
+            ${until.tablet} {
+                grid-column-gap: 0px;
+
+                grid-template-columns: 1fr; /* Main content */
+                grid-template-areas:
+                    ${hasPreFurniture ? `'preFurniture'` : `'media'`}
```
[the remark this answers is not in the document]
Comment (rcrphillips) [reply src/web/layouts/StandardLayout.tsx]: Good catch. Forgot that it had to be (preFurniture AND other template area) OR other template area!

[the remark this answers is not in the document]
Comment (rcrphillips) [reply src/web/layouts/StandardLayout.tsx]: Should be good to review again. Although I still can't work out the nicest indentation..... @gtrufitt

**Review by gtrufitt — APPROVED** [review]
LGTM. Couple of things:

- Would we expect a story to show this difference?
- There's a gap on the left of the score card that isn't there on Frontend, is that fixed in a different PR?

**Comment by rcrphillips** [comment]
@gtrufitt I'll make sure to update the fixture and the story properly in the other PR.

**Pull request merged**